Tests/Microsoft.Teams.Api.Tests/Activities/Message/MessageActivityTests.cs

```diff
@@ -220,4 +221,56 @@ public void JsonDeserialize_Derived()
 
         Assert.Equivalent(expected, activity);
     }
+
+    [Fact]
+    public void JsonSerialize_WebChat_AllowsEmptyConversationType()
+    {
+        MessageActivity activity = new MessageActivity()
+        {
+            Id = "1",
+            ChannelId = new ChannelId("webchat"),
+            From = new Account()
+            {
+                Id = "1",
+                Name = "test",
+                Role = Role.User
+            },
+            Conversation = new Api.Conversation()
+            {
+                Id = "1"
+            },
+            Recipient = new Account
+            {
+                Id = "2",
+                Name = "test-bot",
+                Role = Role.Bot
+            }
+        };
+        string json = JsonSerializer.Serialize(activity, new JsonSerializerOptions { WriteIndented = true, DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull });
+        string exptected = File.ReadAllText(@"../../../Json/Activity/Message/MessageActivity_webChat.json");
+        Assert.Equal(exptected, json);
+    }
+
+    [Fact]
+    public void JsonDeserialize_WebChat_AllowsEmptyConversationType()
+    {
+        var json = File.ReadAllText(@"../../../Json/Activity/Message/MessageActivity_webChat.json");
+        var activity = JsonSerializer.Deserialize<MessageActivity>(json);
+        Assert.NotNull(activity);
```

Comment:
I prefer explicit asserts, rather than Assert.Equivalent, especially when dealing with null/empty values. 

For consistency we can update all the other Assertions in this file (or even this project)